### ISSUE TITLE
Call of duty 2 graphics issues 

### ISSUE THREAD

**Issue by Levan7** [PCSX2/pcsx2#3138]
Using v1.5.0-dev-3309-g5b6b38670 of PCSX2 speed settings are default (2)

Getting graphical issues when starting the game, if i run it in hardware mode (the dx11 just runs out of memory, the open gl doesn't).

![aa](https://user-images.githubusercontent.com/5770076/67427087-75442d00-f5ec-11e9-8e87-3f766974c4b2.gif)
[emuLog.txt](https://github.com/PCSX2/pcsx2/files/3764085/emuLog.txt)
[snaps.zip](https://github.com/PCSX2/pcsx2/files/3764086/snaps.zip)

**Comment by kozarovv** [comment]
This bug affect other Treyarch PS2 games. Spiderman 2, COD2/3, Ultimate SpiderMan. Spiderman 3 texturing issue seems to be related, but partially fixable by enabling frame buffer conversion, maybe is good hint to start searching cause for other games there.

It looks little bit different on every title, but seems to be the same issue. Iirc there was idea that somehow mipmaping is cause of that, at least for Spiderman2.

**Comment by atomic83GitHub** [comment]
No spiderman 3 is unrated as it's not a treyarch games and use a inhouse engine by Vicarious.

**Comment by kozarovv** [comment]
Right, looks like Spiderman3 use some Vicarious proprietary engine, so is unrelated.

**Comment by JeBobs** [comment]
Can confirm the same thing on g9c65b73ae (on both DX11/OGL) with CoD 3, testing on a GTX 1660 ti, Vega 10, and R9 380. Setting it to software fixes the texture issues, but the game becomes unplayable.

**Comment by Shoegzer** [comment]
Confirmed here too, as tested against commit d05b31e91ce815cb92a3d23a6a8d9d566df27857.  This is by far the worst performing ps2 game of those I've tested - average 60% speed in OGL HW mode and 45% speed in OGL SW mode on a reasonably modern system.  The serious graphics issues with menus and ingame as others have reported related to HW modes still occur as well.  It's a shame for such a great and high-profile title in the ps2 library but understandable if the underlying issues are complex (timing related?).

Test system is liunx mint 19.3 and kernel 5.3 using pcsx2-flatpak self-compiled with gcc 9.3.0.  Nvidia gtx 1070M with 440.82 driver, and an Intel i7-7820HK.

@refractionpcsx2 or others - I'm assuming this is a well-known issue but any thoughts on why this title is so challenging would be interesting.
